### ISSUE TITLE
A11y: Set default H3 header level for va-alert CTA widget (general) and correct header level for va-alert components in PW portfolio, where missing

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-cms#13860]
### User story

**AS A** Veteran
**I WANT** va-alert components to have proper headings in hierarchy
**SO THAT** they are accessible to my screen reader.

### Engineering notes / background

We migrated our CTA widgets to `va-alert` components in PR# [24042](https://github.com/department-of-veterans-affairs/vets-website/pull/24042/files#diff-d5b65c152d1c2f1423699e682c6dc0a8a0ce7b79671804092db5201fb0e8c878R17-R20). Unlike with old react component, the heading has to be passed into this one.

E.g., [Refill prescriptions](https://www.va.gov/health-care/refill-track-prescriptions/) – see that the header level on the CTA alert is `<hundefined>`.
<img width="451" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/4054752/f20ee5d1-d76a-434f-a17c-65d6811ccf28">

Identified via https://github.com/department-of-veterans-affairs/va.gov-team/issues/58433#issuecomment-1563727946
Related slack discussion: https://dsva.slack.com/archives/C52CL1PKQ/p1685128804266039

> The component is currently assuming that the CTA widgets are passing in a header but not all of them do.  What needs to happen is that we make H3 the fallback in case no heading is passed, and then it will be up to the code owners of all of those widgets to decide whether or not they need to pass in a header
> we own the base CTA component, but we are not code owners on hardly any of the widgets that consume the component

## Acceptance criteria

- [x] Update CTA component to default to H3 when a heading level is not passed in
- [ ] Unit tests implemented on CTA component to test fallback behavior
- [x] All va-alert components in PW portfolio have a heading level explicitly passed in
  - [x] Cerner widgets (5)
  - [x] BTSSS
  - [x] Prescription refill
  - [x] Secure messaging
  - [x] Appointment scheduling
  - [x] Medical records
  - [x] Test results
